.github/workflows/ci-e2e-cassandra.yml

```diff
@@ -45,6 +50,9 @@ jobs:
     - name: Run cassandra integration tests
       id: test-execution
       run: bash scripts/cassandra-integration-test.sh ${{ matrix.version.major }} ${{ matrix.version.schema }} ${{ matrix.jaeger-version }}
+      env:
+        CASSANDRA_CREATE_SCHEMA: ${{ matrix.create-schema == 'auto' && true || false }}
```

Comment:
this should be not defined in the workflow, but in the shell script. The workflow is just an interface to the test shell script. The shell script is the interface to the actual test. I should be able to run the shell script regardless of this workflow, and if I pass it SKIP_APPLY_SCHEMA=true I expect the script to do the right thing.

Reply:
I see. I have updated it. Thank you.